src/web/components/Nav/Nav.tsx

```diff
@@ -90,7 +90,7 @@ export const Nav = ({ format, nav, subscribeUrl, edition }: Props) => {
                         var showMoreButton = document.getElementById('${showMoreButtonId}')
                         var veggieBurger = document.getElementById('${veggieBurgerId}')
                         var expandedMenuClickableTags = document.querySelectorAll('.selectableMenuItem')
-                        var expandedMenu = document.getElementById('expanded-menu')
+                        var expandedMenu = document.getElementById('expanded-menu-root')
```

Comment:
2 `div`s had `expanded-menu` as an ID. I separated the parent `div` to have suffix `-root`